mslib/msui/constants.py

```diff
@@ -37,8 +37,9 @@
 if '://' in MSUI_CONFIG_PATH:
     try:
         _fs = fs.open_fs(MSUI_CONFIG_PATH)
-    except fs.errors.CreateFailed:
         _fs.makedirs(MSUI_CONFIG_PATH)
+    except fs.errors.CreateFailed:
+        logging.error('FS url "%s" create dir nor supported', MSUI_CONFIG_PATH)
```

Comment:
The logging message does not describe the same error case as `fs.errors.CreateFailed`. The error is described here: https://docs.pyfilesystem.org/en/latest/reference/errors.html#fs.errors.CreateFailed AFAICT it can be emitted when `fs.open_fs` fails for some reason and is not necessarily related to if the FS supports creating directories.